extensions/BMO/template/en/default/pages/etiquette.html.tmpl

```diff
@@ -18,129 +18,107 @@
-  the [% terms.bugs %] you want fixed. Therefore, you should not act as if you
-  expect someone to fix a [% terms.bug %] by a particular date or release.
-  Aggressive or repeated demands will not be received well and will almost
-  certainly diminish the impact and interest in your suggestions.
+    <span class="heading">No obligation</span>.
+    "Open Source" is not the same as "the developers must do my bidding."
+    Everyone here wants to help, but no one else has any <i>obligation</i> to fix
+    the [% terms.bugs %] you want fixed. Therefore, you should not act as if you
+    expect someone to fix a [% terms.bug %] by a particular date or release.
+    Aggressive or repeated demands will not be received well and will almost
+    certainly diminish the impact and interest in your suggestions.
+  </li>
+  
+  <li>
+    <span class="heading">No spam</span>.
+    Posting comment spam will lead to the suspension of your account.
   </li>
 
   <li>
-  <span class="heading">No abusing people</span>.
-  Constant and intense critique is one of the reasons we build great products.
-  It's harder to fall into group-think if there is always a healthy amount of
-  dissent. We want to encourage vibrant debate inside of the Mozilla
-  community, we want you to disagree with us, and we want you to effectively
-  argue your case. However, we require that in the process, you attack
-  <i>things</i>, not <i>people</i>. Examples of things include: interfaces,
-  algorithms, and schedules. Examples of people include: developers,
-  designers and users. <b>Attacking a person may result in you being banned
-  from [% terms.Bugzilla %].</b>
+    <span class="heading">No pointless comments</span>.
+    Limit comments on a [% terms.bug %] to information which will help with 
+    resolving it. Unless requested, additional "I see this too" or "It works for me" 
+    comments are unnecessary unless they are on a different platform or a significantly 
+    different build. Constructive and helpful thoughts unrelated to the topic of 
+    the [% terms.bug %] should go in the appropriate
+     <a href="http://www.mozilla.org/about/forums/">discussion forum</a>.
   </li>
 
   <li>
-  <span class="heading">No private email</span>.
-  Unless the [% terms.bug %] owner or another respected project contributor has asked you
-  to email them with specific information, please place all information
-  relating to [% terms.bugs %]
-  in the [% terms.bug %] itself. Do not send them by private email; no-one else can read
-  them if you do that, and they'll probably just get ignored. If a file
-  is too big for [% terms.Bugzilla %], add a comment giving the file size and contents
-  and ask what to do.
+    <span class="heading">No private email</span>.
+    Do not send comments to [% term.bugs %] by private email; no-one else can read
+    them if you do that, and they'll be missed and/or ignored. If an attachment
```

Comment:
I think you changed the meaning of this (which admittedly wasn't super clear). I believe the point is that you shouldn't email Bugzilla users regarding bugs. This now sounds like you shouldn't email bugs themselves. Also no hyphen in "no one".